tests/integrationv2/utils.py

```diff
@@ -67,7 +86,14 @@ def invalid_test_parameters(*args, **kwargs):
             return True
 
     # Prevent situations like using X25519 with TLS1.2
-    if curve is not None and protocol is not None and curve.min_protocol > protocol:
+    if curve is not None:
+        if protocol is not None and curve.min_protocol > protocol:
             return True
 
+    if provider is OpenSSL:
+        if get_flag(S2N_OPENSSL_VERSION) != "openssl-1.1.1":
+            if protocol is not None and protocol is Protocols.TLS13:
+                # TLS1.3 is only supported by OpenSSL 1.1.1 and greater
```

Comment:
Why does TLS1.3 require OpenSSL 1.1.1 or greater? Can't it use lower versions, just without some options (like no RSA)?

Reply:
https://wiki.openssl.org/index.php/TLS1.3

We only test with openssl 1.1.1, openssl 1.0.2, and openssl 1.0.2-fips.